plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/internal/RegistryActivator.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.registry.internal;
+
+import org.eclipse.core.runtime.Plugin;
+import org.jkiss.dbeaver.utils.RuntimeUtils;
+import org.jkiss.dbeaver.utils.SystemVariablesResolver;
+import org.jkiss.utils.CommonUtils;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+public class RegistryActivator extends Plugin {
+
+    public RegistryActivator() {
+    }
+
+    @Override
+    public void start(BundleContext context) throws Exception {
+        super.start(context);
+
+
+        setJnaNativePath(context);
+    }
+
+    private static void setJnaNativePath(BundleContext context) {
+        /*if (Platform.getOS().equals(Platform.OS_WIN32)) */
+        {
+            // Set JNA library path (#19735)
+            String installPath = SystemVariablesResolver.getInstallPath();
+            Path pluginsPath = Path.of(installPath).resolve("plugins");
+            Bundle jnaBundle = Arrays.stream(context.getBundles()).filter(b -> b.getSymbolicName().equals("com.sun.jna"))
+                .findFirst().orElse(null);
```

Comment:
`Bundle#getSymbolicName` may return `null`. Consider swapping operands around:

```suggestion
            Bundle jnaBundle = Arrays.stream(context.getBundles()).filter(b -> "com.sun.jna".equals(b.getSymbolicName()))
                .findFirst().orElse(null);
```

And a nitpick regarding code style:

```suggestion
            Bundle jnaBundle = Arrays.stream(context.getBundles())
                .filter(b -> "com.sun.jna".equals(b.getSymbolicName()))
                .findFirst().orElse(null);
```